google/cloud/storage/oauth2/google_credentials_test.cc

```diff
@@ -528,18 +528,29 @@ TEST_F(GoogleCredentialsTest, LoadP12Credentials) {
   ScopedEnvironment adc_env_var(GoogleAdcEnvVar(), filename.c_str());
 
   auto creds = GoogleDefaultCredentials();
-  if (creds.status().code() == StatusCode::kInvalidArgument &&
-      absl::StrContains(creds.status().message(), "error:0308010C")) {
-    // With OpenSSL 3.0 the PKCS#12 files may not be supported by default.
-    (void)std::remove(filename.c_str());
-    GTEST_SKIP();
+  EXPECT_EQ(0, std::remove(filename.c_str()));
+
+  if (creds.status().code() == StatusCode::kInvalidArgument) {
+    if (absl::StrContains(creds.status().message(), "error:0308010C")) {
+      // With OpenSSL 3.0 the PKCS#12 files may not be supported by default.
+      GTEST_SKIP();
```

Comment:
Optional:  Stream something enlightening into the `GTEST_SKIP()`s?

Reply:
Done.